packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -0,0 +1,214 @@
+%global nvidia_tesla_470_version 470.82.01
+%global spdx_id %(bottlerocket-license-tool -l $PWD/rpmbuild/BUILD/Licenses.toml spdx-id nvidia)
+%global license_file %(bottlerocket-license-tool -l $PWD/rpmbuild/BUILD/Licenses.toml path nvidia -p ./licenses)
+
+Name: %{_cross_os}kmod-5.10-nvidia
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: NVIDIA drivers for the 5.10 kernel
+# We use these licences because we only ship our own software in the main package,
+# each subpackage includes the LICENSE file provided by the run scripts
+License: Apache-2.0 OR MIT
+URL: http://www.nvidia.com/
+
+# NVIDIA .run scripts from 0 to 199
+Source0: https://us.download.nvidia.com/tesla/%{nvidia_tesla_470_version}/NVIDIA-Linux-%{_cross_arch}-%{nvidia_tesla_470_version}.run
+
+# Common NVIDIA conf files from 200 to 299
+Source200: nvidia-tmpfiles.conf.in
+Source202: nvidia-dependencies-modules-load.conf
+
+# NVIDIA tesla conf files from 300 to 399
+Source300: nvidia-tesla-tmpfiles.conf.in
+Source301: nvidia-tesla-build-config.toml.in
+Source302: nvidia-tesla-path.env.in
+Source303: nvidia-ld.so.conf.in
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}kernel-5.10-archive
+
+%description
+%{summary}.
+
+%package tesla-470
+Summary: NVIDIA 470 Tesla driver
+Version: %{nvidia_tesla_470_version}
+License: %{spdx_id}
+Requires: %{name}
+
+%description tesla-470
+%{summary}
+
+%prep
+# Extract nvidia sources with `-x`, otherwise the script will try to install
+# the driver in the current run
+sh %{S:0} -x
+
+%global kernel_sources %{_builddir}/kernel-devel
+tar -xf %{_cross_datadir}/bottlerocket/kernel-devel.tar.xz
+
+%build
+pushd NVIDIA-Linux-%{_cross_arch}-%{nvidia_tesla_470_version}/kernel
```

Comment:
nit: we could remove these pushd & popd calls, and potentially fold the subpackage into the main package based on today's conversation, since it seems like we'll only ship the "production" Tesla drivers.

Reply:
on second thought - let's leave this scaffolding in place in case we end up wanting to ship multiple versions.